index.html

```diff
@@ -2,11 +2,21 @@
 <html lang="en">
 <head>
     <meta charset="utf-8">
-    <title>Delta Chat Invitation</title>
+    <title>Connect on Delta Chat</title>
+    <meta name="description" content="Delta Chat is a messaging app that works over e-mail.">
+
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="apple-itunes-app" content="app-id=1459523234">
     <!-- todo generic meta tags -->
 
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Connect on Delta Chat">
+    <meta property="og:image" content="https://invite.delta.chat/social.png">
+    <meta property="og:url" content="https://i.delta.chat">
```

Comment:
what is og:url for?

Reply:
could setting it cause problems? (because parameters are not set in that url)?

Reply:
It seems like only Whatsapp sets it to the actual URL https://github.com/deltachat/invite/issues/6#issuecomment-1352646786

But I'm not sure what it does, and I guess it could be taken out. I default to setting more properties with OG stuff.

Reply:
hm, documentation say, that this is the "canonical url". i think, we have to try it out on the major platforms we want to support.